bin/build-module.js

```diff
@@ -30,20 +30,7 @@ function buildModule(filepath) {
   // All external modules are assumed to be CommonJS, and therefore should
   // be skipped by Rollup. We may revisit this later.
   var depsToSkip = Object.keys(topPkg.dependencies || {});
-  depsToSkip = depsToSkip.concat([
-    'crypto', 'fs', 'events', 'path'
-  ]);
-
-  if (pkg.name === 'pouchdb-for-coverage') {
-    // special case - for the coverage reports, the whole thing is
-    // bundled into one index.js. so we don't want to externalize
-    // the pouchdb repos
```

Comment:
I'm not sure why we were doing this, but it turns out we don't need to. I checked it out, and we're still getting 100% code coverage, although will need to check that we're not ignoring more LoC now.